lib/internal/streams/readable.js

```diff
@@ -196,7 +196,8 @@ function Readable(options) {
   Stream.call(this, options);
 
   destroyImpl.construct(this, () => {
-    maybeReadMore(this, this._readableState);
+    if (!options || !options.manualStart)
```

Comment:
I'm ok with `manualStart` given prior art.

Reply:
> What about?
> 
> ```js
> const myStream = createStream()
> myStream.pause()
> ```

Wait, just tested it and maybe you are right, just a second